pkg/rpc/client/rpc.go

```diff
@@ -113,6 +240,30 @@ func (c *Client) GetNEP5Transfers(address string) (*result.NEP5Transfers, error)
 	return resp, nil
 }
 
+// GetPeers returns the list of nodes that the node is currently connected/disconnected from.
+func (c *Client) GetPeers() (*result.GetPeers, error) {
+	var (
+		params = request.NewRawParams()
+		resp   = &result.GetPeers{}
+	)
+	if err := c.performRequest("getpeers", params, resp); err != nil {
+		return resp, err
+	}
+	return resp, nil
+}
+
+// GetRawMemPool returns the list of unconfirmed transactions in memory.
+func (c *Client) GetRawMemPool() ([]util.Uint256, error) {
+	var (
+		params = request.NewRawParams()
+		resp   = new([]util.Uint256)
+	)
+	if err := c.performRequest("getrawmempool", params, resp); err != nil {
```

Comment:
And here is where the bug actually happens after the `*[]`->`[]` change, you have to pass `&resp` here, otherwise you're gonna be returning empty slice for any request.